db/schema.rb

```diff
@@ -43,6 +43,7 @@
     t.integer  "facility_id",            limit: 4
     t.integer  "affiliate_id",           limit: 4
     t.string   "affiliate_other",        limit: 255
+    t.string   "outside_contact_info",   limit: 255
```

Comment:
I suspect whichever of these PRs goes in last will need to get updated to deal with merge conflicts.